core/src/main/java/nu/yona/server/subscriptions/service/UserDto.java

```diff
@@ -68,12 +77,12 @@ public UserDto(@JsonProperty("firstName") String firstName, @JsonProperty("lastN
 			@JsonUnwrapped UserPrivateDto privateData)
 	{
 		this(null, Optional.empty(), null, firstName, lastName, emailAddress, mobileNumber, false /* default value, ignored */,
-				privateData);
+				privateData, Collections.emptySet());
```

Comment:
Should devices not be passable with the JSON creator?

Reply:
I'm working on that part right now.